docs/en/stack/ml/nlp/ml-nlp-deploy-models.asciidoc

```diff
@@ -130,14 +130,30 @@ NOTE: Since eland uses APIs to deploy the models, you cannot see the models in
 {kib}, wait for automatic synchronization, or use the
 {kibana-ref}/machine-learning-api-sync.html[sync {ml} saved objects API].
 
-When you deploy the model, it is allocated to all available {ml} nodes. The
-model is loaded into memory in a native process that encapsulates `libtorch`,
-which is the underlying machine learning library of PyTorch.
-
-TIP: You can set additional deployment options that affect the performance of
-the model. For example, specify the number of allocations and the threads per
-allocation to optimize the throughput and speed of inference requests in your
-production environment. For more information about these options, refer to the
+When you deploy the model, its allocations are distributed across available {ml} 
+nodes. Model allocations are independent units of work for NLP tasks. Throughput 
+can be scaled by adding more allocations to the deployment, it increases the 
+number of {infer} requests that can be performed in parallel. All allocations 
+assigned to a node share the same copy of the model in memory. The model is 
+loaded into memory in a native process that encapsulates `libtorch`, which is 
+the underlying {ml} library of PyTorch.
+
+You can set the number of allocations and the number of threads an allocation 
+uses for your deployment that affect model performance.
+
+The number of allocations setting represents the amount of model allocations 
+across all the {ml} nodes. Increasing the number of allocations generally 
+increases the throughput.
+
+NOTE: Model allocations are distributed in such a way that the total number of 
+used threads does not exceed the allocated processors of a node.
+
+The threads per allocation setting represents the number of threads used by each 
+model allocation during {infer}. Increasing the number of threads generally 
+increases the speed of {infer} requests. The value of this setting must not 
+exceed the number of available allocated processors per node.
+
+Refer to the
```

Comment:
> Refer to the

Since you've covered all the necessary info here now, and since this API is already linked at the beginning of this section, I don't think the link is needed here at the end anymore.